config/example.yaml

```diff
@@ -23,14 +23,14 @@ grains:
   threshold_std_dev: 1.0
   threshold_absolute_lower: 1.0
   threshold_absolute_upper: 1.0
-  absolute_area_threshold: 
+  absolute_area_threshold:
     upper: [500,800] # above surface [Low, High] in nm^2 (also takes null)
     lower: [null, null] # below surface [Low, High] in nm^2 (also takes null)
   direction: upper # Options: upper, lower, both (defines whether to look for grains above or below thresholds or both)
   background: 0.0
 grainstats:
   run: true # Options : true, false
-  cropped_size: 40 # Length (in nm) of square cropped images (can take -1 for grain-sized box)
+  cropped_size: 40.0 # Length (in nm) of square cropped images (can take -1 for grain-sized box)
```

Comment:
I changed to `40.0` (pending SI unit you're implementing) as I thought it would be feasible to have a length in fractions of `nm`.

The typehint for `cropped_size` in `topostats.grainstats.Grainstats()` class is `float` so I made the configuration validation consistent with that. In order to pass validation the value in the configuration file also had to be a float and `type(40)` returns `int` whilst `type(40.0)` returns `float` hence the change here.